web/pom.xml

```diff
@@ -306,26 +306,48 @@
          </plugin>
 
          <plugin>
-            <groupId>org.codehaus.mojo</groupId>
-            <artifactId>exec-maven-plugin</artifactId>
-            <version>1.3.2</version>
+            <groupId>com.github.eirslett</groupId>
+            <artifactId>frontend-maven-plugin</artifactId>
```

Comment:
Could you please very shortly explain, what the plugin basically does? It's just I have never used it before, nothing wrong with the usage at all :)